docs/en/getting-started/quickstart/python/adk/quickstart_test.py

```
@@ -0,0 +1,76 @@
+                [sys.executable, "quickstart.py"],
+                capture_output=True,
+                text=True,
+                timeout=120  # 2 minute timeout
+            )
+            
+            actual_output = result.stdout
+            stderr_output = result.stderr
+            
+            print("    quickstart_test.py:30: --- SCRIPT OUTPUT ---")
+            if actual_output:
+                for line in actual_output.split('\n'):
+                    if line.strip():
+                        print(f"        {line}")
+            else:
+                print("        (No output)")
+            
+            # Check return code
+            if result.returncode != 0:
+                self.fail(f"Script execution failed with return code {result.returncode}")
+            else:
+                print("    quickstart_test.py:32: ✅ PASSED: Script ran successfully and produced output.")
+            
+            # Check output content
+            if not actual_output.strip():
+                self.fail("Script ran successfully but produced no output.")
+            
+            # Check for essential keywords
+            golden_file_path = Path("../../golden.txt")
+            
+            if golden_file_path.exists():
+                try:
+                    with open(golden_file_path, 'r') as f:
+                        keywords = [line.strip() for line in f.readlines() if line.strip()]
+                    
+                    for keyword in keywords:
+                        if keyword in actual_output:
+                            print(f"    quickstart_test.py:41: ✅ INFO: Found keyword '{keyword}' in output.")
+                        else:
+                            print(f"    quickstart_test.py:43: ⚠️ INFO: Did not find keyword '{keyword}' in output.")
+                            
+                except Exception as e:
+                    print(f"    quickstart_test.py:45: ⚠️ WARNING: Could not read golden.txt: {e}")
```

We should use assert statements for test fails instead of print statements in tests